### PR TITLE
Add anniversary banner.

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

Adds an anniversary banner component. Part of a waterfall PR. 

## Todo
- In the next PR I need to implement the persistent state of the 'closed' banner.
- Remove on hover text underline on the Tweet button

https://user-images.githubusercontent.com/31534717/165561625-c9dea667-e002-426b-9e1d-ba5909bb6298.mov

https://user-images.githubusercontent.com/31534717/165561659-3143ebe2-2041-4cb2-887b-7f2e29fe3c5d.mov

### CODE REVIEW

File: src/custom/components/SideBanner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+import SVG from 'react-inlinesvg'
+import { CloseIcon, ExternalLink } from 'theme'
+
+const WIDTH = 440
+const HEIGHT = 440
+const ANNIVERSARY_TWEET_TEMPLATE =
+  'Holy CoW, today @MEVprotection is 1 year old! Check out their UI growth over time to see the CoW-evolution https://youtu.be/nxU11DmBVMk'
+
+export interface BannerProps {
+  type: string
```
Comment (W3stside) [inline src/custom/components/SideBanner/index.tsx:16]: make this type an enum/type

File: src/custom/components/SideBanner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+}
+
+const Banner = styled.div<{ isActive: boolean }>`
+  position: fixed;
+  width: ${`${WIDTH}px`};
```
Comment (W3stside) [inline src/custom/components/SideBanner/index.tsx:21]: no need for double template strings
```suggestion
  width: ${WIDTH}px`;
```

File: src/custom/components/SideBanner/index.tsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  height: ${`${HEIGHT}px`};
```
Comment (W3stside) [inline src/custom/components/SideBanner/index.tsx:22]:
```suggestion
  height: `${HEIGHT}px`;
```

File: src/custom/pages/App/index.tsx [hunk 4]
```diff
@@ -92,6 +94,7 @@ export default function App() {
     <>
       <RedirectAnySwapAffectedUsers />
       <Wrapper>
+        <SideBanner type={'anniversary'} />
```
Comment (W3stside) [inline src/custom/pages/App/index.tsx:97]: can we load this as well with suspense and include in the body?

Comment (fairlighteth) [reply src/custom/pages/App/index.tsx:97]: Yes, done in the other PR now.

**Review by anxolin — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @fairlighteth , the banner itself looks good to me. 
The only thing I'd like to mention is that the banner is displayed above the app menu in a mobile view
image
Is it OK?

**Comment by alfetopito** [comment]
Superseded by #475

**Pull request closed without merging**